games/t.yaml

```
@@ -98,6 +99,26 @@
   video:
     youtube: cU6LL5BXi5E
 
+- name: TekwarGDX
+  originals:
+  - TekWar
+  type: remake
+  repo: 'https://gitlab.com/m210/TekwarGDX'
+  url: 'http://m210.duke4.net/'
+  development: very active
+  status: playable
+  content: commercial
+  lang:
+  - Java
+  framework:
+  - LibGDX
+  license:
+  - GPL3
```

This doesn't look GPL3 compatible
https://gitlab.com/m210/TekwarGDX/blob/master/buildlic.txt
```suggestion
  - Custom
```

---

In that case, Chocolate Duke3D, Duke3D, Duke3d_win32, EDuke32, JFDuke3D, xDuke, NBlood, PCExumed and Rednukem are not GPL2 compatible either, because they have this file too.

---

https://en.wikipedia.org/wiki/Build_(game_engine)#Other_game_ports
yeah seems that's true, good to know

---

OK, so what to do? Have here `Custom ` and then change it to `Custom ` for every other Build game in a separate PR?

---

yep that sounds good